### DISCUSSION TITLE
[HELP]Account suspended

### DISCUSSION THREAD

**Discussion by hobertjing** [community/community#141580] [New to GitHub]
Hi,

I registered an account a few days ago, when i logged in today found that account has been suspended, and i don't know why.

I never push codes after registered, when I try to use the new account, it suspended.
please help me.

Ticket ID: 3050189

**Comment by ghostinhershell** [comment]
Hi @hobertjing, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
